packages/wells/src/client/model/MeasurementType.ts

```diff
@@ -1,4 +1,4 @@
-export const enum MeasurementType {
+export enum MeasurementType {
   GammaRay = 'GammaRay',
   Caliper = 'Caliper',
   Resistivity = 'Resistivity',
```

Comment:
if we use `export const enum MeasurementType`, we cannot import the enum in a js project. If we use `export enum MeasurementType` we can import in a js project, but we cannot import internally from another module. I solved this by creating an internal `MeasurementType` for the tests, since import failed. Let me know if you have a good workaround.